src/lib.rs

```diff
@@ -288,16 +293,24 @@ pub mod pallet {
                 .try_into()
                 .map_err(|_| Error::<T>::NameTooLong)?;
 
+            let mut bounded_participants = BoundedVec::<
```

Comment:
Curious if this couldn't be just `BoundedVec::<T::AccountId, T::MaxPoolParticipants>`